test/integration_test.py

```diff
@@ -757,7 +757,7 @@ def _list_sources(http_client: RequestMethods):
                        sourceId=tdr_client.lookup_source_id(source_spec))
             for source_spec in (
                 TDRSourceSpec.parse(source).effective
-                for source in config.tdr_sources(catalog)
+                for source in config.source(catalog)
```

Comment:
Running the IT should have caught this, no?

```suggestion
                for source in config.sources(catalog)
```

Reply:
Could you respond to my question, please?